ci/Jenkinsfile

```diff
@@ -241,7 +274,11 @@ timeout(time: 2, unit: 'HOURS') {
                     ['backend/target/release/backend'],
                     ['backend']
                 ),
-                'cargo-clippy/doc': runDockerPostgresSidecar('cargo clippy && cargo doc'),
+                'cargo-clippy-doc': runDockerPostgresSidecar(
+                    'cargo clippy && cargo doc --document-private-items',
```

Comment:
Is there a specific reason why `make doc-backend` couldn't be used?

Reply:
I wrote it because in most cases in the already existing file, "cargo" commands were used and not the Makefile, so not really